client/src/components/QModal.jsx

```diff
@@ -29,9 +29,9 @@ function QModal({ pID }) {
     //close window on submit
     alert(`${aModal ? 'Answer' : 'Question'} Submitted!`);
 
-    axios.get(`/a/questions/${pID}?count=${100}`)
-      .then((res) => res.data)
-      .catch((err) => { console.log(err); });
+    axios.get(`a/questions/${pID}?count=${100}`)
```

Comment:
I believe the forward slash is required in front of this url. If it isn't, ignore this comment

Reply:
request works! :)